lib/Net/SAML2/Protocol/Artifact.pm

```diff
@@ -13,10 +11,6 @@ with 'Net::SAML2::Role::ProtocolMessage';
 
 # ABSTRACT: SAML2 artifact object
 
-=head1 NAME
```

Comment:
Any particular reason you are removing NAME?

Reply:
Yeah, because ABSTRACT adds it as well: https://metacpan.org/release/TIMLEGGE/Net-SAML2-0.78/view/lib/Net/SAML2/Protocol/Artifact.pm You see it being there twice. Without NAME it is only going to be there once.